app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/dialogs/VoiceSearchWidget.java

```diff
@@ -261,7 +261,7 @@ public void onRequestPermissionsResult(int requestCode, String[] permissions, in
                 show();
 
             } else {
-                super.show();
+                super.show(true);
```

Comment:
We may want to change the true to a enum in the future to improve readability (as a follow-up)